src/config/workload.rs

```diff
@@ -180,6 +188,10 @@ impl Keyspace {
         self.vkind.unwrap_or(ValueKind::Bytes)
     }
 
+    pub fn value_compression_ratio(&self) -> f64 {
```

Comment:
Is it worth documenting this in one of the example config files?

Reply:
Will update the example configs